health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/configs/UICustomizations.js

```
@@ -0,0 +1,193 @@
+      data.params.limit = 100;
+      data.params.offset = 0;
+      cleanObject(data.body.ProjectStaff);
+      return data;
+    },
+    additionalCustomizations: (row, key, column, value, t, searchResult) => {
+      switch (key) {
+        case "ACTIONS":
+          let options = [
+            { code: "1", name: "VIEW_DASHBOARD" },
+            { code: "2", name: "VIEW_CUSTOM_REPORT" },
+          ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)